src/lua/help_en_US.lua

```diff
@@ -1,16 +1,16 @@
---- help (en_US)
+--- help (en_US) 2.x
 
 return {
     help = [[
-To get help, see the Tarantool manual at https://tarantool.org/doc/
-To start the interactive Tarantool tutorial, type 'tutorial()'
+To get help, see the Tarantool manual at https://tarantool.io/en/doc/
+To start the interactive Tarantool tutorial, type 'tutorial()' here.
 
 Available backslash commands:
 
-  \set language <language> for setting language
-  \set delimiter <delimiter> for setting delimiter
-  \help show this screen
-  \quit quit interactive console
+  \set language <language>   -- set language (lua or sql)
+  \set delimiter <delimiter> -- set end-of-line delimiter
```

Comment:
Here I still not sure it is correct to name it 'end of line' delimiter. Maybe commands delimiter?

Reply:
We decided to use 'expression delimiter' term here.